src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -201,13 +201,16 @@ To define a _docker-run option_, use the generic information *docker-stop-option
 
 For instance by using the docker-stop-options generic information with the value --time 30, the container will be stopped after 30s.
 
+TIP:  a parameter *docker.file.keepcontainer* is defined in the dockerfile script engine properties file (in config/scriptengines/ folder). Put the value to *yes* if you do not want to stop the container.
```

Comment:
@cdiop @fviale Is it the right place to put this info here? As this feature is not yet implemented as a generic information in the front end, should it be moved to the Admin guide or somewhere?